api/activity.go

```diff
@@ -121,7 +121,7 @@ type ActivityIssueStatusUpdatePayload struct {
 }
 
 type ActivityPipelineTaskStatusUpdatePayload struct {
-	TaskId    int        `json:"taskId"`
+	TaskID    int        `json:"taskID"`
```

Comment:
json:"taskID"

This will translate to the json field. Do we have a reference to decide whether json field should be taskId or taskID ?

Reply:
JSON:API mentions that 
> [Member names SHOULD be camel-cased](https://jsonapi.org/recommendations/#:~:text=Member%20names%20SHOULD%20be%20camel%2Dcased). 

But there is no further information about whether to use `taskId` or `taskID`.

According to golang/lint#124, `ID` is the correct style. If we use `ID` in golang, I think it would be more consistent to also use `ID` in json fields. 

IMO, it's weird to have
 ```
TaskID int `json:"taskId"`
```